### DISCUSSION TITLE
Does the library support Expo?

### DISCUSSION THREAD

**Discussion by appsgenie** [calintamas/react-native-toast-message#425] [General]
Hi, 
Do we know if this library could be used with react-native on Expo?
I did not see any native customizations so I assume it will work?

**Comment by ViniDevBR** [comment]
Yes, It works, i'm using this now on my project using EXPO GO

**Reply by appsgenie** [reply]
do you know if it works with a custom dev client build? not Expo Go.

**Comment by suprunchuk** [comment]
Yes. This is still working in 2024. You can use it safely.

**Reply by appsgenie** [reply]
So this means works without expo plugins?

**Reply by suprunchuk** [reply]
> So this means works without expo plugins?

it's just a package that can work with or without Expo
